zerver/webhooks/bitbucket3/tests.py

```diff
@@ -8,6 +8,22 @@ class Bitbucket3HookTests(WebhookTestCase):
     EXPECTED_TOPIC = "sandbox"
     EXPECTED_TOPIC_BRANCH_EVENTS = "sandbox / {branch}"
 
+    # Diagnostics Events:
+    def test_ping(self) -> None:
+        expected_message = "Congratulations! The Bitbucket Server integration has been successfully set up!"
+        self.send_and_test_stream_message("diagnostics_ping",
+                                          "Bitbucket Server Ping",
+                                          expected_message,
+                                          HTTP_X_EVENT_KEY="diagnostics:ping")
+
+    def test_ping_with_user_defined_topic(self) -> None:
+        self.url = self.build_webhook_url(topic="my topic")
+        expected_message = "Congratulations! The Bitbucket Server integration has been successfully set up!"
```

Comment:
Make this "Congratulations! The Bitbucket Server webhook was configured successfully!" :)

Reply:
Sure thing! Made the change.